dev/preview/workflow/preview/deploy-monitoring-satellite.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```

Comment:
This file is was moved from .werft/observability/install-satellite.sh and modified slightly.